pkgs/development/python-modules/pwinput/default.nix

```
@@ -0,0 +1,32 @@
+{
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  lib,
+}:
+buildPythonPackage rec {
+  pname = "pwinput";
+  version = "1.0.3";
+
+  pyproject = true;
+
```

The chance is pretty low that somebody will test the module on Python releases which are EOL.

```suggestion
  pyproject = true;

  disabled = pythonOlder "3.7";
```